cockatrice/src/dlg_edit_user.cpp

```diff
@@ -34,8 +34,8 @@ DlgEditUser::DlgEditUser(QWidget *parent, QString email, int gender, QString cou
     int i = 1;
     foreach(QString c, countries)
     {
-        countryEdit->addItem(QPixmap(":/resources/countries/" + c + ".svg"), c);
-        if(c == country)
+        countryEdit->addItem(QPixmap(":/resources/countries/" + c.toLower() + ".svg"), c);
+        if (c == country)
```

Comment:
Here the value of "country" arrives from the server, sowe probably need to ensure it's lowercase (or make the test case insensitive)